### ISSUE TITLE
Library imports Angular directly resulting in a bloated bundle

### ISSUE THREAD

**Issue by x87** [vsternbach/angular-ts-decorators#43]
Hey,

I've found a problem recently. My application is built and delivered as a single bundle with no AngularJS inside, My requirement is that AngularJS has to be included on the same page as a separate bundle. 

When I'm using `angular-ts-decorators` I see that AngularJS is included in my app bundle which is unexpected behaviour. This happens due to this lines:

```
import { bootstrap, element, injector, isFunction, module as module$1 } from 'angular';
import * as angular from 'angular';
```

Not sure why would you need both lines, but anyway the result is that AngularJS is packed into the same bundle as dependency.

I would like to see that `angular-ts-decorators` has a prerequisite requirement to have AngularJS be available globally on the page. Without AngularJS being available on the page, there is no point in using decorators anyway. The same approach is used for example in the Angular Material library.
Basically you could import needed functions from the global variable `window.angular`.

**Comment by vsternbach** [comment]
It kind of should import angular as dependency, are you using webpack for bundling?
If yes, then why angular is defined as global? Ayhow, you can configure your webpack setup to treat angular as global, so it won't be included in your build

**Comment by x87** [comment]
Ok, I've solved my issue using webpack's `externals` config. Thanks for the tip.
